src/app/Page/main-view/main-view.component.ts

```diff
@@ -14,12 +13,28 @@ import { EmployeesService } from 'src/app/Services/employees.service';
 })
 export class MainViewComponent implements OnInit {
 
+  employees: EmployeeStatus[] = [{
```

Comment:
in a configurable app, these would come from the database, you could model them in EF with relationships rather than calc on the front end.